app/slashcommands/command_invite.go

```diff
@@ -38,137 +37,193 @@ func (*InviteProvider) GetCommand(a *app.App, T i18n.TranslateFunc) *model.Comma
 	}
 }
 
-func (*InviteProvider) DoCommand(a *app.App, c request.CTX, args *model.CommandArgs, message string) *model.CommandResponse {
+type multiResponses struct {
+	responses []string
+}
```

Comment:
I'm curious what made you choose to put this `[]string` into a field in a struct and put the `append`, `len`, and `strings.Join` into methods?

Reply:
I initially thought I needed more data than just the final string message and wanted to encapsulate that logic somewhere else to hide the repetition. But in the current implementation, it doesn't add much, probably it only adds clutter.